resilience4j-spring-boot-common/build.gradle

```diff
@@ -3,8 +3,8 @@ dependencies {
 
     compileOnly(libraries.spring_boot2_aop)
     compileOnly(libraries.spring_boot2_actuator)
-    annotationProcessor(libraries.spring_boot2_config_processor)
-    annotationProcessor(libraries.spring_boot2_autoconfigure_processor)
+    compileOnly(libraries.spring_boot2_config_processor)
```

Comment:
Why does that change behavior? Do you know it?

Reply:
If we add `spring-boot-autoconfigure-processor` as annotationProcessor, it makes `spring-autoconfigure-metadata.json` and `spring-configuration-metadata.json`.
![image](https://user-images.githubusercontent.com/19386038/70786053-1e2e2d80-1dcf-11ea-9c18-1d430f8bba71.png)

The one produced by the spring-boot-common module and the one created by the downstream module(like spring-boot, spring-boot2) are duplicated. So I changed spring-boot-common module not to make those files.

Reply:
But can't we then delete the dependencies?

Reply:
Yes you are right. I removed them